patches/server/0817-Fix-CraftCriteria-defaults-map.patch

```diff
@@ -0,0 +1,37 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jake Potrebic <jake.m.potrebic@gmail.com>
+Date: Mon, 4 Oct 2021 22:31:51 -0700
+Subject: [PATCH] Fix CraftCriteria defaults map
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/scoreboard/CraftCriteria.java b/src/main/java/org/bukkit/craftbukkit/scoreboard/CraftCriteria.java
+index 22801e33fa15322c37cd11d73a40a43fa721a8e4..7241e0900df6e5dedf08db2fc88211b1256ed648 100644
+--- a/src/main/java/org/bukkit/craftbukkit/scoreboard/CraftCriteria.java
++++ b/src/main/java/org/bukkit/craftbukkit/scoreboard/CraftCriteria.java
+@@ -37,7 +37,13 @@ final class CraftCriteria {
+     }
+ 
+     static CraftCriteria getFromNMS(Objective objective) {
++        if (CraftCriteria.DEFAULTS.containsKey(objective.getCriteria().getName())) { // Paper - fix criteria defaults
+         return CraftCriteria.DEFAULTS.get(objective.getCriteria().getName());
```

Comment:
What type is DEFAULTS? Surely you could `Objects.requireNonNullElseGet` here?

Reply:
ya, you are correct.